external-crates/move/crates/move-compiler/src/parser/ast.rs

```diff
@@ -658,7 +660,13 @@ pub enum Exp_ {
     // e.f
     Dot(Box<Exp>, Name),
     // e.f(earg,*)
-    DotCall(Box<Exp>, Name, Option<Vec<Type>>, Spanned<Vec<Exp>>),
+    DotCall(
+        Box<Exp>,
+        Name,
+        /* is_macro */ bool,
```

Comment:
Would it be better to record this as `Option<Loc>`, a la `mut`?

Reply:
Yeah...

Reply:
Just sad because of the amount of plumbing to do lol